src/core/ast.ml

```diff
@@ -135,6 +136,7 @@ type token =
 	| Question
 	| At
 	| Dollar of string
+	| Spread
```

Comment:
How does this differ from `Unop Spread`? Is this for parsing the function signature?

Reply:
Yes, it's because `...` can be found not only at expression level, but also in arguments lists and function signatures.